.github/workflows/merge.yml

```diff
@@ -110,6 +110,10 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
 
+      - name: Run autonomi tests
+        timeout-minutes: 25
+        run: cargo test --release --package autonomi --lib --features="full,fs"
+
       - name: Run autonomi tests
```

Comment:
i think the same workflow is repeated twice! **Run autonomi tests**
Maybe some rebase conflicts?

Reply:
oh cool!